.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## ✍️ Proposed changes
+
+Describe the big picture of your changes here and communicate why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+🎟 *Jira ticket:* [Name of ticket](https://my.jira/ticket)
+
+
+## 🛠 Types of changes
+
+What types of changes does your code introduce? _Put an `x` in the applicable boxes._
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+
+## ✅ Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/mongodb/leafygreen-ui/blob/master/CONTRIBUTING.md) doc
+- [ ] Tests pass locally with my changes
+- [ ] I ran `npm run prettier` before committing my changes.
```

Comment:
This should go away when we integrate a CI.

Reply:
Are you saying that the CI will run prettier, or check that prettier has been run? I _think_ you mean the latter.

Either way, yes.

Reply:
The latter, yes.